src/modules/map/components/olMap/index.js

```diff
@@ -1,4 +1,8 @@
 import { OlMap } from './OlMap';
+import { BaOverlay } from './handler/measure/BaOverlay';
 if (!window.customElements.get(OlMap.tag)) {
 	window.customElements.define(OlMap.tag, OlMap);
+}
+if (!window.customElements.get(BaOverlay.tag)) {
+	window.customElements.define(BaOverlay.tag, BaOverlay);
```

Comment:
Can we do the registration also within the handler? A handler should be like a plug-and-play mechanism via DI without anny other dependencies.

Reply:
Sure